src/seedu/addressbook/parser/Parser.java

```diff
@@ -73,40 +78,44 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
```

Comment:
Indentation seems to be wrong. Please check coding standard.

Reply:
Noted. Thank you! I will update in the level3/4.